# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/identity/moz.build
# Commit: 459739221a88
# Full Hash: 459739221a884b2b1c2adab42fd19fafe279bd20
# Author: Sandor Molnar <smolnar@mozilla.com>
# Date: 2025-06-05 04:07:37
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing build bustages
#   
#   This reverts commit ab681e36aeafc21ba62bb4680bdaa6206f02546d.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r fc9986230203 -r 459739221a88 dom/credentialmanagement/identity/moz.build
--- a/dom/credentialmanagement/identity/moz.build	Thu Jun 05 00:15:19 2025 +0300
+++ b/dom/credentialmanagement/identity/moz.build	Thu Jun 05 00:19:41 2025 +0300
@@ -13,22 +13,16 @@
     "IdentityCredentialSerializationHelpers.h",
     "IdentityNetworkHelpers.h",
     "NavigatorLogin.h",
-    "WebIdentityChild.h",
-    "WebIdentityHandler.h",
-    "WebIdentityParent.h",
 ]
 
 IPDL_SOURCES += [
-    "PWebIdentity.ipdl",
+    "IPCIdentityCredential.ipdlh",
 ]
 
 UNIFIED_SOURCES += [
     "IdentityCredential.cpp",
     "IdentityNetworkHelpers.cpp",
     "NavigatorLogin.cpp",
-    "WebIdentityChild.cpp",
-    "WebIdentityHandler.cpp",
-    "WebIdentityParent.cpp",
 ]
 
 include("/ipc/chromium/chromium-config.mozbuild")